index.html

```diff
@@ -1 +1,133 @@
+<html lang="en">
+<head> 
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <main id="main-doc">
+         <nav id = "navbar">
+            <header >programming languages documentation</header>
+            <a class = "nav-link" href="#introduction">introduction</a>
+            <a class = "nav-link" href="#what_you_should_already_know">what you should already know</a>
+            <a class = "nav-link" href="#javascript_and_java">javascript and java</a>
+            <a class = "nav-link" href="#Hello_World">Hello World</a>
+            <a class = "nav-link" href="#variables">variables</a>
+            <a class = "nav-link" href="#declaring_variables">declaring variables</a>
+            <a class = "nav-link" href="#variable_scope">variable scope</a>
+            <a class = "nav-link" href="#global_variables">global variables</a>
+        </nav>
+        <section id = "introduction"class = "main-section">
+            <header >
+             <h1>introduction</h1>
+            </header>
+            <ul>
+                <li>main1</li>
+                <li>main2</li>
+            </ul>
+            <p>JavaScript is a cross-platform, object-oriented scripting language. It is a small and lightweight language.</p>
+            <p>Inside a host environment (for example, a web browser), JavaScript can be connected to the objects of its environment to provide programmatic control over them. </p>
+
+        </section>
+        <section id = "what_you_should_already_know"class = "main-section">
+            <header >
+                <h1>what you should already know</h1>
```

Comment:
please only use one h1 tag on your page for SEO. consider using lower title tags like h2 for lower-level titles